### PR TITLE
Fix/metis

### PR DESCRIPTION
**Pull request by serbanmaerean**
remover version of the cmake dependency.

### CODE REVIEW

**Comment by adamjstewart** [comment]
Can you give a little more explanation of why this change is necessary? I just looked at the `CMakeLists.txt` for the latest release and 2.8 is indeed the minimum version required:
```
cmake_minimum_required(VERSION 2.8)
```
If you're building metis 4, perhaps all you need to do is add the cmake dependency for older versions as well?

File: var/spack/repos/builtin/packages/metis/package.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Metis(Package):
     variant('int64', default=False, description='Sets the bit width of METIS\'s index type to 64.')
     variant('real64', default=False, description='Sets the bit width of METIS\'s real type to 64.')
 
-    depends_on('cmake@2.8:', when='@5:', type='build')
+    depends_on('cmake', type='build')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/metis/package.py]: i agree with @adamjstewart, you probably want to keep the line untouched and add
```
depends_on('cmake', when='@:4.999', type='build')
```

**Comment by adamjstewart** [comment]
Actually, Metis 4 doesn't even use Cmake. So I'm not sure what was wrong with the original package.

**Comment by alalazo** [comment]
The line you are changing was modified in da8fd5b30a696e5e7db91b929e840cb5e1bd070b and ddaba07d8297544f2587f0c31fcbd2dafa824eb3 . Adding the authors of these commits as reviewers.

**Comment by alalazo** [comment]
@goxberry I don't know why but I can't request you as a reviewer, so making the call in a comment.

**Comment by serbanmaerean** [comment]
I am using cmake 3.8.  It tried to use cmake 2.8 instead.  All I care is about using the latest version of cmake.

**Comment by adamjstewart** [comment]
@serbanmaerean 
```python
depends_on('cmake@2.8:')
```
means it requires CMake 2.8 **or newer**, so you can use CMake 3.8 with this as well.

**Comment by serbanmaerean** [comment]
Yup, it works now, not sure what was going on before.  Sorry for the spam.

**Comment by adamjstewart** [comment]
No worries, glad it's working for you!

**Pull request closed without merging**